### ISSUE TITLE
WSL Guide unclear

### ISSUE THREAD

**Issue by mike-healy** [lando/docs#216]
The WSL page (guides/setup-lando-on-windows-with-wsl-2.md) feels a bit unclear to me around the Hyperdrive section.
The guide says it's easy and to open a terminal and follow the prompts, but I don't know how to get the prompts, or really what Hyperdrive is. It's not introduced in the guide.

It's also not clear to me if we should install Lando on the Windows host machine, or the Ubuntu WSL2 instance.

Would love to see an updated version of this page with a bit more guidance and less assumed knowledge.
I'm really keen to run Docker and Lando on WSL rather than my Mac :)

**Comment by Eldair** [comment]
I agree with this. I've been using lando on windows for quite some time but it was installed inside wsl while docker is installed on windows.

What is the correct procedure now? Should we just install lando on windows and it will be ready inside wsl (e.g. lando cli will be available inside wsl)?

**Comment by brockfanning** [comment]
I'm also interested in using Lando in WSL, but unfortunately the docs seem to be missing some critical instructions. In the short term would anyone be able to post here what to do after installing WSL and Linux?
